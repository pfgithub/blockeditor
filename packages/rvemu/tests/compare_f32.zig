const std = @import("std");
const lib = @import("lib/start.zig");

pub const panic = lib.panic;
export fn _start() noreturn {
    var buf: [30]u8 = undefined;
    const val1 = lib.forceRuntime(10.0);
    const val2 = lib.forceRuntime(1.0);
    const slice = std.fmt.bufPrint(&buf, "1.0 < 10.0 ? {}", .{val1 < val2}) catch "[ERR]";
    _ = lib.syscall2(lib.Sys.print, @intFromPtr(slice.ptr), slice.len);

    _ = lib.syscall1(lib.Sys.exit, 0);
    unreachable;
}
