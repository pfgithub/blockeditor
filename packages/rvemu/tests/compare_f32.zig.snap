[LOG] 1.0 < 10.0 ? true
Exited with code 0
